packages/utils-expression-evaluator/test/integration/evaluators/Evaluator-test.ts

```diff
@@ -2,11 +2,11 @@ import type { ActorExpressionEvaluatorFactory } from '@comunica/bus-expression-e
 import type { ITermFunction, MediatorFunctionFactory } from '@comunica/bus-function-factory';
```

Comment:
I initially assumed these test files would remain in the jest package, so that they would get published to npm, and could also be used outside of this monorepo.
Unless you see a good reason not to do this?

Reply:
Resolved ofline